app/src/main/java/ru/vssemikoz/newsfeed/models/NewsItem.java

```diff
@@ -68,6 +74,8 @@ public void setImageUrl(URL imageUrl) {
     public void setPublishedAt(Date publishedAt) {
         this.publishedAt = publishedAt;
     }
+
```

Comment:
Please, try to avoid unnecessary empty lines. They're placed at the end of the file and between code parts. But they shouldn't follow 2 and more in a row.